initfiles/componentfiles/configschema/xsd/topology.xsd

```diff
@@ -55,7 +55,6 @@
                             <xs:attribute name="prefix" type="xs:string" hpcc:displayName="Prefix" hpcc:tooltip="Need one"/>
 
                         </xs:complexType>
-                        alias="" name="thor400_44_sla" prefix="thor400_44_sla"
```

Comment:
Unrelated, please address with separate Jira

Reply:
@RussWhitehead Addressed. Please review the update.